src/main/java/gregtech/api/gui/ModularUI.java

```diff
@@ -188,6 +193,11 @@ public Builder bindPlayerInventory(InventoryPlayer inventoryPlayer, TextureArea
             return bindPlayerInventory(inventoryPlayer, imageLocation, 8, 84);
         }
 
+        //to be called in order to offset the player inventory from the top of the window
+        public Builder bindPlayerInventory(InventoryPlayer inventoryPlayer, TextureArea imageLocation, int yOffset) {
+            return bindPlayerInventory(inventoryPlayer, imageLocation, 8, 84 + yOffset);
+        }
```

Comment:
To be consistent, the non offset version of this method should redirect to the offset version of the method, to match the changes to `defaultBuilder`